Packs/Campaign/Scripts/DBotGroupPhishingIncidents/DBotGroupPhishingIncidents.py

```diff
@@ -0,0 +1,39 @@
+from CommonServerPython import *
+
+model_name = 'phishing_clustering'
+field_for_grouping = 'emailsubject,emailbody,emailbodyhtml'
```

Comment:
Why taking **emailbody** and **emailbodyhtml** ?  What the difference between them ? Won't it bring redundancy ?

Reply:
usually only one of them exists but I don't know which one of them, so better to show both